src/App.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './css/main.css';
-import { LessonInfo } from './components';
+import HomePage from './components/pages/HomePage';
```

Comment:
Use the destructure import, that's why we have an `index.js`
```suggestion
import { HomePage } from './components';
```